lib/OrdinaryDiffEqVerner/test/ode_verner_tests.jl

```diff
@@ -0,0 +1,182 @@
+@test length(sol1) == length(sol2)
+@test SciMLBase.successful_retcode(sol1)
+@test SciMLBase.successful_retcode(sol2)
+
+# -------------------------------------------------------------
+### Vern8
+println("Vern8")
+dts = (1 / 2) .^ (6:-1:3)
+check_convergence(dts, probnumbig, Vern8(), 8)
+check_convergence(dts, probbig, Vern8(), 8)
+
+tabalg = ExplicitRK(tableau = constructVerner8(BigFloat))
+sol1 = solve(probnumbig, Vern8(); dt = 1 / 2^6, adaptive = false, save_everystep = false)
+sol2 = solve(probnumbig, tabalg; dt = 1 / 2^6, adaptive = false, save_everystep = false)
+@test sol1.u[end] - sol2.u[end] < 1e-10
+
+sol1 = solve(probbig, Vern8(); dt = 1 / 2^3, adaptive = false, save_everystep = false)
+sol2 = solve(probbig, tabalg; dt = 1 / 2^3, adaptive = false, save_everystep = false)
+@test minimum(sol1.u[end] - sol2.u[end] .< 1e-10)
+
+sol1 = solve(prob, tabalg; dt = 1 / 2^6)
+sol2 = solve(prob, Vern8(); dt = 1 / 2^6)
+@test length(sol1) == length(sol2)
+@test SciMLBase.successful_retcode(sol1)
+@test SciMLBase.successful_retcode(sol2)
+
+# -------------------------------------------------------------
+### Vern9
+println("Vern9")
+dts = (1 / 2) .^ (6:-1:3)
+check_convergence(dts, probnumbig, Vern9(), 9)
+check_convergence(dts, probbig, Vern9(), 9)
+
+tabalg = ExplicitRK(tableau = constructVernerEfficient9(BigFloat))
+sol1 = solve(probnumbig, Vern9(); dt = 1 / 2^6, adaptive = false, save_everystep = false)
+sol2 = solve(probnumbig, tabalg; dt = 1 / 2^6, adaptive = false, save_everystep = false)
+@test abs(sol1.u[end] - sol2.u[end]) < 1e-15
+
+sol1 = solve(probbig, Vern9(); dt = 1 / 2^3, adaptive = false, save_everystep = false)
+sol2 = solve(probbig, tabalg; dt = 1 / 2^3, adaptive = false, save_everystep = false)
+@test minimum(abs.(sol1.u[end] - sol2.u[end]) .< 1e-15)
+
+sol1 = solve(probbig, tabalg; dt = 1 / 2^6)
+sol2 = solve(probbig, Vern9(); dt = 1 / 2^6)
+@test length(sol1) == length(sol2)
+@test SciMLBase.successful_retcode(sol1)
+@test SciMLBase.successful_retcode(sol2)
+
+# -------------------------------------------------------------
+### RKV76IIa
+println("RKV76IIa")
+# dts = [1, 0.5, 0.25, 0.125, 0.0625, 0.03125, 0.015625, 0.0078125, 0.00390625, 0.001953125, 0.0009765625, 0.00048828125, 0.000244140625, 0.0001220703125, 0.00006103515625]
+dts = [2, 1, 0.5, 0.25]
+
+check_convergence(dts, prob_oop, RKV76IIa(), 7)
+# check_convergence(dts, prob_iip, RKV76IIa(), 7)
+
+tabalg = ExplicitRK(tableau = constructVernerEfficient7(BigFloat))
+sol1 = solve(probnumbig, RKV76IIa(); dt = 1 / 2^6, adaptive = false, save_everystep = false)
+
+sol2 = solve(probnumbig, tabalg; dt = 1 / 2^6, adaptive = false, save_everystep = false)
+@test abs(sol1.u[end] - sol2.u[end]) < 1e-10
+
+sol1 = solve(probnumbig, RKV76IIa(); dt = 1 / 2^3, adaptive = false, save_everystep = false)
+
+sol2 = solve(probnumbig, tabalg; dt = 1 / 2^3, adaptive = false, save_everystep = false)
+diff = sol1.u[end] - sol2.u[end]
+@test minimum(abs.(diff) .< 1e-10)
```

Comment:
should probably just remove the other solve and this check since it won't tableau match